### ISSUE TITLE
DNDi Open Synthesis Network

### ISSUE THREAD

**Issue by bendndi** [OpenSourceMycetoma/Series-1-Fenarimols#13]
The first two participating institutions from the DNDi Open Synthesis Network have recieved the preliminary data for the MycetOS project and ahole to initiate work over the Summer:

Illinois Mathematics ans Science Academy (IMSA) https://www.imsa.edu/site_section/discover/
Augusta University https://www.augusta.edu/

Hopefully representatives from both locations will join in on our Github discussions soon!

**Comment by bendndi** [comment]
The key files, including the OSN wanted laist, can be found in the Files section of Series-1-Fenarimols:
https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/OSN_P4_Master%20target%20list%2020190418.sdf
https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/OSN%20P4%20Mycetos%20full%20data%20set%2020190417.sdf

**Comment by johnimsa** [comment]
I’m John Thurmond, a faculty member at the Illinois Mathematics and Science Academy (IMSA), and have been speaking with Ben from DNDi about joining this project.  IMSA students will start with the syntheses of compounds P4_A_001, P4_A_003, P4_A_006, P4_B_003, and P4_B_004 from the OSN P4 Master target list.

**Comment by mattodd** [comment]
This is great! Welcome on board @johnimsa ! As an open source project, it'd be very good if your students were able to host their work publicly on a lab notebook so that the community can see what's going on. Is that possible? You have probably discussed all this with Ben. Can you briefly describe how you might go about working? Do you think your students will be happy to engage in conversations here on this platform so that we can all be connected?

@bendndi I downloaded the sdf but Chemdraw won't open it. Do you get the same error? If yes, perhaps you have to zip the file before upload? I don't know. Also, the target list: are you able to describe how the targets were derived? If that (and/or the structures) could be put somewhere (e.g. [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/wiki/Current-Targets-and-Activities) or [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/wiki/Desirable-Compounds-Not-Yet-Synthesised)) then we'd be sure we're not duplicating. I know it all takes time... Over in OSM we tried tagging Issues that describe active syntheses with the label "Being Synthesised Now" to try to allow easy collation of molecules under active investigation, but the system only works as well as the people maintaining the tags!

**Comment by johnimsa** [comment]
Thank you for the welcome @mattodd , we're very excited about being part of the community.  The structures of the compounds we are going to synthesize are P4_A_001, P4_A_003, P4_A_006, P4_B_003, and P4_B_004.  We opened the sdf file with DataWarrior.  We're starting with the hydroxy intermediate provided by Ben to prepare these few target compounds to get familiar with the chemistry and we'll be following lab notebook procedures posted in this project. We'll then be able to prepare more compounds in the future (fall semester).   We will be able to host our work publicly in some form of lab notebook to share our procedures and data.

![image](https://user-images.githubusercontent.com/49991699/56857615-8100b080-6935-11e9-99bb-6cf06e4a58be.png) starting intermediate

![image](https://user-images.githubusercontent.com/49991699/56856750-be117680-6926-11e9-8539-f871d88e9350.png)  P4_A_001               
![image](https://user-images.githubusercontent.com/49991699/56856754-d4b7cd80-6926-11e9-8b57-a256457baad9.png)  P4_A_003            
![image](https://user-images.githubusercontent.com/49991699/56856756-e00af900-6926-11e9-9e7a-d9b6f8f3769f.png)   P4_A_006             
![image](https://user-images.githubusercontent.com/49991699/56856771-324c1a00-6927-11e9-8d39-6392ef29b581.png)  P4_B_003                
![image](https://user-images.githubusercontent.com/49991699/56856774-42fc9000-6927-11e9-97d2-59b1d20d5233.png)  P4_B_004

**Comment by bendndi** [comment]
@mattodd Thanks for pointing out the issue with the sdf files, I saw the same problem. This confirms what I'd seen earlier which is that Github does not support sd as a format..

I was however able to get around this by uploading as a zip file [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/OSN_P4%20MycetOS%2020190429.zip)

If you can check if that works for you, that would be great. The three main files are in there, the OSN master target list (molecules I came up with and which I pulled from some of the suggestions on MycetOS e.g. from @MFernflower) plus the data sets in sdf format for use in designing molecules.

**Comment by bendndi** [comment]
@mattodd here is an image of the OSN target list, with IMSA selected compounds highlighted
![OSN compounds 20190429a](https://user-images.githubusercontent.com/36627399/56887135-0b016400-6a71-11e9-9bed-0bc5187351f5.png)

**Comment by MFernflower** [comment]
@bendndi this is AMAZING! however I'd recommend you remove the compound with the N-Si (E.G the one without the methyl spacer) bond because of stability concerns

**Comment by MFernflower** [comment]
@bendndi What is the process of depositing compounds into this database?

**Comment by MFernflower** [comment]
@johnimsa please spin this off to a new ghi when you have time

**Comment by bendndi** [comment]
> @johnimsa please spin this off to a new ghi when you have time

@MFernflower I'm not sure what you mean by this, can you please clarify :-)

**Comment by bendndi** [comment]
> @bendndi this is AMAZING! however I'd recommend you remove the compound with the N-Si (E.G the one without the methyl spacer) bond because of stability concerns

Will do

**Comment by bendndi** [comment]
> @bendndi What is the process of depositing compounds into this database?

@MFernflower just send me ideas or post ideas on here and I will add them in the the master list. Whenever the master list is updated (i.e. new ideas, successful completon of synthesis, or new assigning of compounds to synthetic teams,) I update it and swap the new version in for the old! The master sd list contains a column which records the source of the original idea.

**Comment by bendndi** [comment]
> it'd be very good if your students were able to host their work publicly on a lab notebook

@mattodd do you have a specific lab note book you could recommend to @johnimsa ? The DNDi Open Synthesis Netowrk don't require any kind of specific lab notebook use, w ejust run with the expectation that at the time anything gets written up for publication each participant can provide the necessary experimental details.

Alos one other thing I was wondering about... IMSA have NMR and FTIR capabilities but no Mass spec capabilities. Do you have MS (or bettr still LCMS) capabiliites at UCL, and if so could we perhaps set up an arrangment where IMSA send samples to UCL for LCMS, and you them send them on to Wendy for test?

**Comment by MFernflower** [comment]
IR spec and NMR should be good enough to confirm if team made the product
yes? @bendndi

**Comment by bendndi** [comment]
> IR spec and NMR should be good enough to confirm if team made the product yes?

@MFernflower  Not really... in my experience / opinion MS is pretty much essential. MS + 1HNMR as a minimum at all places I've worked. IR and NMR are entireley qualitative and open to interpretation, MS has an irrefutable quantitative element.

**Comment by MFernflower** [comment]
One last thing - I had a csv list of molecules I proposed - I feel if you merge that with your masterlist we would get a nice backlog of compounds for teams allover the world to dig into

https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/Analog%20Planning/mfernflower/proposed_mols_datawarrior.csv




@bendndi

**Comment by MFernflower** [comment]
@johnimsa Some quick searches from chemical suppliers to help in your efforts:

https://www.sigmaaldrich.com/catalog/product/aldrich/415308?lang=en&region=US monoethylpiperazine is rather cheap so I think this would be better than the ELN route of Cl-intermediate --> couple to monoBOCpiperazine ---> deprotect ---> couple to ethyl bromide

The furnished piperazinyl urea fragment is a bit pricey and only available from oakwood chemicals so this isn't gonna help 

1-(2-Methoxyphenyl)piperazine is rather cheap from SA (sub 40 usd for amounts needed) and so is 1-(2-Fluorophenyl)piperazine

**Comment by mattodd** [comment]
Hi @johnimsa great, looks good. Everyone: it'd help the community to have these structures tagged as "being synthesized now". Over in Open Source Malaria we experimented with this by having a Github issue for each such molecule carrying a tag saying just that. That works where students can take charge of their molecule and post updates and plans to the issue as a form of diary, which remains open during the synthesis attempt. An alternative is to insert the molecules into the MycetOS Google Sheet Master List of compounds and tag them there as "being synthesized" which could involve the inclusion of a web link back here. A lengthier way is to post them to a "Desirable Compounds Not Yet Synthesised" page of the wiki, which installs them as "desirable and in progress" ([here](https://github.com/OpenSourceMalaria/Series4/wiki/Desirable-Compounds-Not-Yet-Synthesised)'s the one for OSM Series 4). Any preferences? This is what @MFernflower is talking about when he mentions starting a new "ghi" (=Git Hub Issue) but I think we could usefully think about how we want to do this first. It will impact others we hope to work with here ping @alintheopen

@bendndi when I unzip your zip file on a Mac I get an odd file extension "OSN_P4 MycetOS 20190429.zip.cpgz" which, when I rename to .sdf does not open properly. Help @drc007?

@bendndi the image you posted of the targets seems to have that functionality of tagging something as "ongoing" plus a person responsible. What's the software?

@bendndi Lab notebooks. An ongoing, annoying issue. Labarchives has a free trial version which might do. Other solutions require some kind of inputs (cost or running a server, e.g. CDD, chemotion). The SGC guys use Zenodo as a way to deposit Word and PDF files. I don't like it as a solution since it doesn't work like a lab notebook with respect to raw data like spectra and images. Interestingly Github itself can be used as a public file store, which would work, but it's non-trivial. _We need to fix this together by getting some money together for software development!_

@bendndi @johnimsa Yes we can run LCMS in my lab of samples. If they become numerous we may need to think about cost, but we can certainly get started and see if it works.

@bendndi reopening this issue, by the way, I think you closed it by accident. I'm going to alert the community to your lovely pic above of the current targets ping @dmitrij176

**Comment by MFernflower** [comment]
@mattodd

Re the cpgz file:

"MacOS method of unzipping a zip file is to first create a new file with
the zip.cpgz extension before actually returning the contents of the ZIP
archive. It then opens this CPGZ archive and re-creates the ZIP file again,
then decompress it gives back into zip.cpgz file... and this goes on
infinitely.

This may happen when macOS doesn't understand the type of ZIP compression
used with the file, so it thinks that you want to compress the file instead
of decompressing it. Since CPGZ is the default format used for compression,
which is Compressed UNIX CPIO Archive file, the file is just being
compressed and decompressed over and over again.

The solution is to use a third party tool such as PeaZip or 7zip"

**Comment by drc007** [comment]
@mattodd the file is actually a folder containing three files, I've emailed them to you.

**Comment by MedChemProf** [comment]
I have a couple of new students beginning in the lab in a few weeks. Is the process to contributing compounds to the MycetOS project just picking targets on the Master Target List that are labeled as 'Not Started' and then making them in the lab?

**Comment by MFernflower** [comment]
@bendndi @MedChemProf I believe the OSN masterfile is due for an update

**Comment by bendndi** [comment]
@MedChemProf Yep, the èrpcess is simply picking compounds from the master list which are "Not started",  posting the sleection on here (so that I can update the list to reflect that you're owrking on them), and then cracking on with the synthessi.
Note that I can provide some intermediate if needed, see[ elsewhere on this project](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/2#issuecomment-473235937)

@MFernflower I'll update the master list once I'm back in the office later next week.

**Comment by bendndi** [comment]
The Synthetic Schemes for elaboration of the DNDi intermediate to make componuds on the OSN list can be found **[HERE](https://au-mynotebook.labarchives.com/share/Hung%2520Duong/MTguMnwyMDcyMi8xNC9UcmVlTm9kZS80Nzg5MTE4MjB8NDYuMTk5OTk5OTk5OTk5OTk2)** and **[HERE](https://au-mynotebook.labarchives.com/share/Matthew%2520O'Dowd%2520Hons/MC4wfDM2MzA4LzAvVHJlZU5vZGUvNDEyNjE5MjE2fDAuMA==)**

Just check the tabs on the left hand side of the page to see the lab notebook pages of the students from @mattodd lab in Sydney who pioneered these synthetic routes

**Comment by AndyBishopSevenoaks** [comment]
Thanks for getting us involved @mattodd we have a group of students ready to get stuck into this come the new academic year. How would we be able to obtain some intermediate to work from?

**Comment by MFernflower** [comment]
^ Paging @bendndi

**Comment by bendndi** [comment]
@AndyBishopSevenoaks I'd be happy to send on some of the intermediate, I just need the shipping details and your email address - you can either post these here or send on to me at bperry@dndi.org

**Comment by bendndi** [comment]
I've updated the OSN target list to include the compounds from @MFernflower . This can be found as an sdf [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/OSN_P4%20MycetOS%2020190604.zip)

Here is the current target list as an image:
![image](https://user-images.githubusercontent.com/36627399/58856807-eb7ade00-86a3-11e9-970b-9f88abb989d1.png)

**Comment by MFernflower** [comment]
@AndyBishopSevenoaks what compounds from the list do you see your students making?

**Comment by bendndi** [comment]
I've updated the OSN master list for MycetOS to take into account the activites of @luxsika , and also to add in the logDs for the proposed moleucles for comarison with the logD's used in the original [mycetos paper](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0006437) (using same lodG calculation method from Stardrop / optibrium.

![image](https://user-images.githubusercontent.com/36627399/59329382-0a562180-8cef-11e9-8cb5-bb84f2e9e81f.png)

**Comment by bendndi** [comment]
Out of interest, does anyone know if i'ts possible to "pin / sticky" a comment to the top of a forum / issue chat? Would be ideal to have the up to date image of OSN list at the top of this page...

**Comment by MFernflower** [comment]
@bendndi I dont seem to notice any changes in the graphic above? Also I dont think github supports stickies just yet

**Comment by bendndi** [comment]
@MFernflower the image includes logDs calculated by stardrop, and the compounds selected by UCL have been flagged as such.

**Comment by MFernflower** [comment]
Please disregard the comments I made yesterday as I managed to mix up the UCL and imsa groups

**Comment by dmitrij176** [comment]
Good day @bendndi. I have chosen 3 compounds from the OSN master list to be synthesised. These are **P4_C_001, P4_C_002, P4_C_003**.

**Comment by MFernflower** [comment]
@dmitrij176 This lab notebook will be of help https://au-mynotebook.labarchives.com/share/Matthew%2520O'Dowd%2520Hons/MC4wfDM2MzA4LzAvVHJlZU5vZGUvNDEyNjE5MjE2fDAuMA==


I would recommend you go a slightly different route (Perhaps SOBr2 instead of socl2)???

**Comment by luxsika** [comment]
Hi @bendndi 

Previously, I mentioned that I was going synthesize  P4_A_002, P4_A_005 and P4_B_010.  Two of the reagents will only arrive in mid July, so in the meantime, I will be synthesizing P4_A_005 , P4_B_001, P4_C_009.

Hopefully I will be able to synthesize the initial compounds, but I will keep updating here. 
Sorry for any inconvenience. 

Luxsika

**Comment by MFernflower** [comment]
@luxsika Link to your lab notebook?

**Comment by MFernflower** [comment]
@luxsika if you plan to go via the 4-piperidone route for the alcoholic compound you can also knock another compound off the list!

**Comment by MFernflower** [comment]
@dmitrij176 @luxsika This lab notebook will be of help https://au-mynotebook.labarchives.com/share/Matthew%2520O'Dowd%2520Hons/MC4wfDM2MzA4LzAvVHJlZU5vZGUvNDEyNjE5MjE2fDAuMA==


I would recommend you go a slightly different route (SOBr2 instead of SOCL2)???

**Comment by dmitrij176** [comment]
Project update

![image](https://user-images.githubusercontent.com/44785501/61552025-713dc780-aa5f-11e9-9ac2-d76f456734ae.png)

@bendndi @mattodd @MFernflower, I am currently working on 6 compounds that will be finished and fully characterised in early August.
 P4_C_001, P4_C_002, P4_C_003 are from the OSN master list (they appear under 2 codes in the scheme). DM8-DM10 are novel structures.
logD calculations have been performed using ChemAxon at pH 7. For data consistency, @bendndi could you please recalculate logD values with StarDrop (unfortunately, I dont have access to it). Indeed, it appears that there is a certain degree of discrepancy in the generated results. 

SMILES for the related structures:

**DM6-1/ P4_C_001** : Fc1cc(Cl)ccc1C(c1cccnc1)N1CCOCC1

**DM7-1/ P4_C_003** : Fc1cc(Cl)ccc1C(c1cccnc1)N1CCCCC1

**DM8-1** : COCCN(CCOC)C(c1cccnc1)c1ccc(Cl)cc1F

**DM9-1** : CC(C)(C)c1ccc(N(c2ccc(C(C)(C)C)cc2)C(c2cccnc2)c2ccc(Cl)cc2F)cc1

**DM10-1**: Fc1cc(Cl)ccc1C(c1cccnc1)N1CCc2ccccc2C1

**DM11-1/ P4_C_002** : Fc1cc(Cl)ccc1C(c1cccnc1)N1CCC(F)(F)CC1

[Synthesis update.zip](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/3412020/Synthesis.update.zip)

**Comment by MFernflower** [comment]
Compounds 8 and 9 should give interesting test results

**Comment by luxsika** [comment]
Hi all! I just wanted to give a quick update of my research. I have synthesized and purified 5 compounds from the OSN master. They will be shipped off soon for biological evaluation.  I may synthesis one more compound, depending on time as I am only in the lab till End of july/Mid august. My MSc thesis will be due on August 30th, so full characterization be done by then. 
![untitled](https://user-images.githubusercontent.com/50544220/61569030-62660d80-aa7d-11e9-92fb-9beb464d273a.png)


These are the compounds that I have made. @bendndi @MFernflower Would you be able to update the OSN master list please?


Also, I am bit confused. In terms of the OSN master list, how was it generated? and why were these modifications proposed? Would someone be able to send me any papers relating to this? Will be great for my thesis.


Thank you
Luxsika

**Comment by MFernflower** [comment]
@luxsika some structural explanations can be found here: https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/Theses%20Reports%20Docs/Hung%20Duong%20Hons%20Thesis/Thesis%20Presentation%20HPD2017%20v7.pptx

**Comment by MFernflower** [comment]
Great work everyone! This is more a note to me and @bendndi but need to update the open synthesis network image so I can re-edit the wiki

**Comment by bendndi** [comment]
@MFernflower @luxsika @dmitrij176  I've updated the OSN list with status / new additions
![image](https://user-images.githubusercontent.com/36627399/61616517-eb8d6800-ac68-11e9-9ba5-ca560e47922a.png)

**Comment by bendndi** [comment]
@MFernflower @luxsika @dmitrij176 Updated OSN target with new status / new ideas

![image](https://user-images.githubusercontent.com/36627399/61616761-9e5dc600-ac69-11e9-8294-e620a42e5567.png)

**Comment by bendndi** [comment]
@dmitrij176 let me know once your compounds are completed and ready for test and I'll update again.

**Comment by bendndi** [comment]
@dmitrij176 re LogD differences - this is perfectly normal, all LogD algorithms generate logD differently and are known to not necessarily corrlate well. The important thing is to make sure always compare using the same algorithm.

Note the question as to which logP / logD calculation is best has been tackled over and over again by many; the general consensus is that on a global level no one method is better than another, although at the more granular (i.e. chemical series) level you may find one calculation method predicts better than others when comparing to experimental. However, we currently don't have an experimental logD against which to benchmark the various methods...

**Comment by bendndi** [comment]
@luxsika Some pointers on the OSN target list

The OSN master list, at least the DNDi designed target components, are simply a diversity scan of different options around the core, focused mainly on the amino substituent. This is a (the?) classic approach to early hit-to-lead chemistry, particularly in the phenotypic space or when no structural info on the target is available. We're not designing each compound to answer a specific question, they are designed as a whole to anser the question -_What is tolerated, and does anything give us a suprising jump in potency / efficacy?_   If after 50 or 100 compounds investigated in this way we don't find anything that gives us a big (~10x) jump in potency -i.e. if all compounds have about the same potency (flat SAR) or only the hit has potency and the rest are inactive - I'd probably start asking serious questions about if the series has any future potential.

So, the P4_A-xxx compounds are scanning the piperazine nitrogen with _"what about alkyl?, what about Urea?, what about amide?, what about benzyl..."_ 

The P4_B-xxx are a bit more focused - _"we known N-Ph is accepable off the piperazine, so now lets mess with the electronics and the steric space around that ring and see what happens"_

P4_C-xxx are a bit more broad again - _"piperazine is good, so what about morpholine? piperidines?, homopiperazine..."_

Once a project moves further into the Lead optimization space it will become a more focused design-led process, but for now the empirical (scatter-gun) approach is an essential avenue.

**Comment by MFernflower** [comment]
@bendndi when you get a chance can you remove the duplicate firewalker (organosilicon) compound?

**Comment by fantasy121** [comment]
I want to claim compounds P4_D_MXF004, P4_D_MXF005, P4_D_MXF006 and P4_D_MXF009 as part of my synthesis for the coming months. In two days, I will make a more detailed post about all of the molecules I’m planning to synthesise in the early stage of my PhD that I think would be of interest.

**Comment by bendndi** [comment]
@fantasy121 Thakns for letting me know, here is the updated snapshot:
![image](https://user-images.githubusercontent.com/36627399/61859059-aa46c380-aec7-11e9-9985-06db79d9f7a7.png)

**Comment by MFernflower** [comment]
@fantasy121 I assume lithium addition to the relative ketones is going to be the main synthetic route?

**Comment by MFernflower** [comment]
@fantasy121 Added a link to your PHD eln in the wiki

**Comment by fantasy121** [comment]
@MFernflower For the most part I think nBuLi would be the route I’ll be using. For the halogenated versions which I’m planning to synthesise, it is a little bit more tricky. I am having some difficulties in coming up with a  strategy to synthesis the asymmetrically substituted anthracenone (in red) which is a precursor to the final product with the six-member ring lock.
![20190729 GitHub Fused Ring Scheme](https://user-images.githubusercontent.com/36176821/62024065-ec00a280-b216-11e9-914a-7a67dcd3bce3.png)
I played around with a few different combinations, I came up with a possible solution (in the red box), essentially it’s a double Friedels-Craft reaction to get to that ketone. I still feel that I haven’t got to the most reliable route yet though.
![20190729 GitHub 6m Fused Rings](https://user-images.githubusercontent.com/36176821/62024071-f4f17400-b216-11e9-83c4-06c636bddb16.png)

**Comment by fantasy121** [comment]
@MFernflower I've also added a link to my PhD note in the new post I just made. Both notebooks are also hyperlinked together as well so you can easily jump from one to the other

**Comment by MFernflower** [comment]
I really cannot find any of the ketones needed for sale from suppliers other than the Butyrophenone, Fluorenone and Anthrone @fantasy121 How would you plan to make the ethoxy compound?

**Comment by drc007** [comment]
Not an area of chemistry I am familiar with but how stable will the tertiary alcohol be?

![temp](https://user-images.githubusercontent.com/7080516/62026039-c43b1b80-b1d1-11e9-99b2-81d615c79cfc.png)

**Comment by MFernflower** [comment]
Might be wise to just make the unhalogenated six-lock and check stability -
I don't think this mechanism will cause any issues with the five-locks
however! @fantasy121 @drc007

>

**Comment by fantasy121** [comment]
@MFernflower Do you mean the ethoxy as in MXF006? I'm thinking of applying the same nBuLi route I've been using for many previous molecules I made. I would just need to brominate the ethoxy side group first and I have a few candidate for that. If unfortunately nBuLi starts attacking my ether, I've got this back-up plan as well, although this reaction is for diphenyl ethoxy tertiary alcohol. Might need to tweak reaction condition to incorporate that pyridyl group. Maybe down the bromobenzene ratio down to 1 equiv and then adding bromopyridine as 1 equiv ratio after the benzene addition. Also I could shorten the reaction time and/or lower temperature to make the diphenyl species less favourable. It might take some experimentation.
![20190730 Ethoxy Product](https://user-images.githubusercontent.com/36176821/62103922-5a119c00-b2e1-11e9-8f59-f161215962fc.png)

**Comment by fantasy121** [comment]
The paper reporting that Barbier-Grignard reaction:
https://www.sciencedirect.com/science/article/pii/S004040391302128X

**Comment by fantasy121** [comment]
@drc007 I would definitely starting making the unhalogenated version first just to test the robustness of the synthetic route. That incidentally would also give me the product P4_D_MXF009.

**Comment by drc007** [comment]
I think that some of those routes will result in an extra carbon in the side-chain?

**Comment by fantasy121** [comment]
> Might be wise to just make the nonhalogenated six-lock and check stability - I don't think this mechanism will cause any issues with the five-locks however! @fantasy121 @drc007 @mattodd

I would definitely make the unhalogenated version first to test the robustness of the synthetic route. That would also give the product P4_D_MXF009

**Comment by fantasy121** [comment]
> I think that some of those routes will result in an extra carbon in the side-chain?

Thank you. You caught me in a moment of blindness. It seems I've miscounted one of the carbons. It does make thing surprising more affordable though.
![20190730 Ethoxy Product](https://user-images.githubusercontent.com/36176821/62104844-0f455380-b2e4-11e9-84e0-1ccbfa3643af.png)

**Comment by mattodd** [comment]
@drc007 the concern about the elimination you mention above - not sure it's such a risk. The big R group goes axial, to avoid the clash, meaning the OH is pseudo-eq, meaning it's hard to eliminate because of bad overlap of that sigma* with the pi system. Not impossible, just harder than you might think, I'd wager.

**Comment by MFernflower** [comment]
@mattodd I'd love for further MD studies to be done on MXF009 but even a simple tool like molview really shows the clash well: http://molview.org/?q=OC1(C2=CN=CC=C2)C2=CC=CC=C2CC2=C1C=CC=C2

**Comment by dmitrij176** [comment]
@bendndi **DM6-1/ P4_C_001** and **DM7-1/ P4_C_003** have been synthesised, purified and sent for biological evaluation.
Remaining 4 will be finished soon.
Could you please update the OSN master list.

**Comment by luxsika** [comment]
Hi! The compounds that I have synthesized have been sent for biological evaluation. 
![image](https://user-images.githubusercontent.com/50544220/62427422-a77b7800-b6ea-11e9-8a40-06e8d8778040.png)

**Comment by bendndi** [comment]
Uèdated OSN list

![image](https://user-images.githubusercontent.com/36627399/63329578-86de2f80-c332-11e9-8c8d-5f9f9f4c6fbf.png)

**Comment by MFernflower** [comment]
@bendndi can you add C1(CCC1)C(O)(C=1C=NC=CC1)C1CCC1 to the OSN list as MXF_012

I dont know if @fantasy121 is going to try the DCI route to get this molecule or not so i'd list it as not started?

**Comment by MFernflower** [comment]
@bendndi can you add C(C(C1CCC1)(C1=CC=CN=C1)O)1CCC1 to the OSN list as MXF_012

I dont know if @fantasy121 is going to try the DCI route to get this molecule or not so i'd list it as not started?

**Comment by dmitrij176** [comment]
@wwjvdsande the Excel file with the latest information on biological testing of novel Fenarimols is missing data on MIC50. Could you please provide us with that and the method information as well ?
Thank you in advance.

**Comment by bendndi** [comment]
@MFernflower Updated OSN list

![image](https://user-images.githubusercontent.com/36627399/63847409-00f36180-c98e-11e9-9729-cf6850d7971c.png)

**Comment by dmitrij176** [comment]
@bendndi with respect to the OSN list, would it be possible to show which compounds have already been made, including their potencies as well ?
This will keep everyone updated on the latest developments in the chemical library.

**Comment by bendndi** [comment]
@dmitrij176 The compounds in green are the ones which have already been made from the list (white for "not started", orange for "In progress", Green for "Completed". Biodata for these componuds are still under evaluation.

For other compounds in the series (i.e. ones from the series but not made via the OSN) there is a master google docs table [here:](https://docs.google.com/spreadsheets/d/1YhK-3i2KwuVabo1GbZSgVjAUbavEICCMKi5v-EhNq80/edit#gid=954203120) 

and also images [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/tree/master/Biological%20Evaluation)

The data set is also available as an sd file [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/blob/master/OSN_P4%20MycetOS%20lists.zip)

Personally this last file is the best one to use if you're interested in assessing the biodata - just open the sdf file in [datawarrior
](http://www.openmolecules.org/datawarrior/download.html)

**Comment by bendndi** [comment]
Updated OSN list 20Sep 2019

![image](https://user-images.githubusercontent.com/36627399/65309162-d3b06280-db8b-11e9-8708-ebfe4849b346.png)

**Comment by MFernflower** [comment]
@bendndi I came up with another analogue that might be worth adding to the list:

CC(C)(C)OC(=O)N1CC(C1)OC(C1=CN=CC=C1)C1=C(F)C=C(Cl)C=C1

tert‐butyl 3‐[(4‐chloro‐2‐fluorophenyl)(pyridin‐3‐
yl)methoxy]azetidine‐1‐carboxylate

![2390490952](https://user-images.githubusercontent.com/3164942/69515211-d9278180-0f1b-11ea-8f6e-b8c41e94ce62.png)

**Comment by MFernflower** [comment]
Matrix sells needed precursor for like 20 dollars!!! http://www.matrixscientific.com/024669.html

**Comment by MFernflower** [comment]
@bendndi if possible can you update list to mark tested compounds as so and to include the new ideas me and few others were throwing out?

**Comment by dmitrij176** [comment]
MycetOS project update

![Project update](https://user-images.githubusercontent.com/44785501/70398320-13138000-1a12-11ea-8e82-31ea59670544.jpeg)

I would like to reserve the following compounds from the OSN list:

**DM17-1/ P4_C_MXF003**: ClC(C=C1F)=CC=C1C(N2CCC(CC2)=O)C3=CN=CC=C3

**DM18-1/ P4_A_007**: ClC(C=C1F)=CC=C1C(N2CCN(CCN(C)C)CC2)C3=CN=CC=C3

**DM19-1/P4_C_MXF001**: ClC(C=C1F)=CC=C1C(N2N=CC=C2)C3=CN=CC=C3

**DM22-1/ P4_C_MXF002**: ClC(C=C1F)=CC=C1C(N2N=CN=C2)C3=CN=CC=C3

**DM23-1**: CC(C)(OC(N1CC(OC(C2=C(C=C(C=C2)Cl)F)C3=CN=CC=C3)C1)=O)C

@bendndi DM23-1 is a newly proposed analogue by @MFernflower which currently does not have an assigned OSN code. Could you please add the structure to the updated scheme. 

DM4-3B and DM16-1 will be used as positive controls in biological testing.
Unfortunately, DM9-1 synthesis was unsuccessful.
[Project update.zip](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/3937331/Project.update.zip)

**Comment by MFernflower** [comment]
@dmitrij176 I'd assume that DM9 might be too sterically hindered - perhaps making the version with the two tert butyl groups lopped off might be worthwhile if time permits?

**Comment by bendndi** [comment]
@dmitrij176 @MFernflower Thanks, I'll update sometime later this week once I'm back in the office!
B

**Comment by bendndi** [comment]
Hi all, the OSN list is now updated:
![image](https://user-images.githubusercontent.com/36627399/70630754-606b3980-1c2c-11ea-8213-b3f66e75f030.png)
@dmitrij176 @MFernflower

**Comment by MFernflower** [comment]
@bendndi @dmitrij176 OSN update looks good but if I am not mistaken it is missing two compounds? 


![fen2](https://user-images.githubusercontent.com/3164942/70684470-435a5900-1c74-11ea-9386-eaf562d6a08a.png)
![fen1](https://user-images.githubusercontent.com/3164942/70684471-435a5900-1c74-11ea-9111-e99c45609117.png)

**Comment by bendndi** [comment]
Hi all, the OSN list is now updated:

Green - Completed
Orange - synthesis ongoing
White - not started

![image](https://user-images.githubusercontent.com/36627399/70694181-49295c00-1cbf-11ea-917d-c8433cd73914.png)

@MFernflower

**Comment by MFernflower** [comment]
@bendndi Thank you Dr.Perry! Could you upload a CSV file containing all this data for users of datawarrior? I think the old files and the zips on the github are out of date!

**Comment by bendndi** [comment]
@MFernflower Done

**Comment by MFernflower** [comment]
@bendndi Might not be a bad idea to somehow mark the di-tert-butyl as being difficult or impossible (steric bulk?) to synthesize as @dmitrij176 could not complete it

**Comment by dmitrij176** [comment]
MycetOS project update

![Project update Feb 2020](https://user-images.githubusercontent.com/44785501/76029628-c9f99c00-5f2c-11ea-8421-3ab4640e7252.jpeg)


**DM24-1**: ClC(C=C1F)=CC=C1C(NC2=CC=CC=C2)C3=CN=CC=C3
**DM25-1**: ClC(C=C1F)=CC=C1C(N(C)C2=CC=CC=C2)C3=CN=CC=C3

Following a number of attempts to synthesise DM9-1 analog, a series of reactions with aniline based substrates were carried out. The assessment of the following
synthetic route showed that both aniline (DM24-1) and N-methylaniline (DM25-1) can be added to the Fenarimol core. Reaction with diphenylamine in DM26-1 (which is the most structurally related  reagent to the tert-butyl substrate in the original DM9-1 experiment) generated trace amounts of the desired product. The results suggest that extra aromatic rings along with tert-butyl groups do lead to steric hindrance in these reactions.
It was decided not to include DM26-1 in the OSN list as the product could not be characterised due to its very small percentage in the product mixture. Therefore, I would conclude that the outcome of the two aforementioned reactions is the same.

DM23-6 experiment with 1-Boc-3-hydroxyazetidine produced the desired Fenarimol analog after the introduction of several key changes to the original reaction protocol.
The following result was achieved by first deprotonating the substrate with NaH, followed by the addition of DMAP catalyst to promote reaction completion. Under standard conditions (triethylamine and KI catalyst), the product did not form.

@bendndi could you please add DM24-1 and DM25-1 to the OSN list and update the status of the completed molecules.

[Project update March 2020.zip](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/4295745/Project.update.March.2020.zip)

**Comment by MFernflower** [comment]
@dmitrij176 @bendndi I think it would be wise to not add DM26 to the OSN list

Lovely work however!

**Comment by MFernflower** [comment]
@wwjvdsande @Wilson-Lm Just curious how long it takes for the assay to be done - Fungi can have variable growth times from fast to agonizingly slow - I remember having to culture Hericium for a buddy's culinary mushroom farm and it took forever to take hold

**Comment by MFernflower** [comment]
@dmitrij176 When you get a chance can you split the compounds you send to Wendy up into a new issue?

**Comment by bendndi** [comment]
Updated OSN list

![image](https://user-images.githubusercontent.com/36627399/76200828-89cb3f80-61f2-11ea-99de-b1fcd81f8ae0.png)

Great work @dmitrij176 ! I added in the benzyl analogues to DM24 and DM25, could be worth a look too if you have a chance.

**Comment by wwjvdsande** [comment]
> @wwjvdsande @Wilson-Lm Just curious how long it takes for the assay to be done - Fungi can have variable growth times from fast to agonizingly slow - I remember having to culture Hericium for a buddy's culinary mushroom farm and it took forever to take hold

@MFernflower unfortunately, M. mycetomatis is one of those slow growing fungi. One MIC takes 2-3 weeks due to the culture time and we always test in duplicate or triplicate. We will do an initial screening with high concentrations of the compound hopefully in two weeks time from now. We will initiate the starter culture this week. The initial results we will deposit here and those who seem to have some activity will be tested further. In case anybody want to add some compounds in the screen it would be nice if we can add them all in one go.

**Comment by alintheopen** [comment]
Cc @fantasy121

**Comment by fantasy121** [comment]
@wwjvdsande As a point of reference, could you also give everyone here a preferred miminum amount of each compound to send to you for biological evaluation?

**Comment by wwjvdsande** [comment]
@fantasy121 we need at least 2 mg for the initial screenings. If the compound is weighed and the weight is on the tube that would be great. It needs to be able to be dissolved in DMSO or water or culture media for the in vitro screenings. In case a compound is very potent and we want to proceed to larvae work it is preferred if the compound can be dissolved in normal saline, water or PBS. However in case we need to dissolve in DMSO, we can only inject a maximum of a 5% DMSO solution into the larvae.

**Comment by wwjvdsande** [comment]
@fantasy121 we need at least 2 mg for the initial screenings. If the compound is weighed and the weight is on the tube that would be great. It needs to be able to be dissolved in DMSO or water or culture media for the in vitro screenings. In case a compound is very potent and we want to proceed to larvae work it is preferred if the compound can be dissolved in normal saline, water or PBS. However in case we need to dissolve in DMSO, we can only inject a maximum of a 5% DMSO solution into the larvae.

**Comment by wwjvdsande** [comment]
Dear all, due to the COVID-19 outbreak many research facilities in Europe closed their laboratories for now. Also we are working from home. That means that it will take a bit longer before we can test the compounds generated. We hope that the COVID-19 outbreak will not disrupt all our lives to long. When we will return we will start the cultures up and test all compounds received. Any compounds not sent to us yet can be sent then. Madurella is a slow grower it will take at least 2 weeks till a month till it is ready to be used for in vitro testing.

**Comment by dmitrij176** [comment]
@bendndi I have noticed that the OSN list in issue #29 is an outdated version.
The updated scheme https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/13#issuecomment-596427111 with the latest compound status was posted on the 9th of March

**Comment by bendndi** [comment]
> @bendndi I have noticed that the OSN list in issue #29 is an outdated version.
> The updated scheme [#13 (comment)](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/13#issuecomment-596427111) with the latest compound status was posted on the 9th of March

Yep, I've updated it now -I didn0't have the most recent one to hand but I needed to put i a placeholder since @fantasy121 wanted it to be the top commment in the thread he had opened.
Will update again once I've compiled the list.

**Comment by dmitrij176** [comment]
@bendndi apologies for this misunderstanding.
All the compounds from todays update are already in the OSN list. This includes the 2 molecules that are currently in development (DM27-1 and DM28-1). I have double checked the lists, its all correct.
DM4-3B and DM16-1 are positive controls, but DM4-3B has been synthesised before and therefore should not be included.
I am using Chemdraw, so no problems with generating SMILES. Whenever new molecules are being proposed, I will provide the SMILES codes.
In addition to that, would you still prefer me to create a list with all SMILES ?

Lastly, I have noticed that DM24-1/P4_C_DM24 and DM27-1/P4_C_014  structures were drawn incorrectly. Both are missing a H atom. Could you please correct that.

![Untitled Document-1](https://user-images.githubusercontent.com/44785501/81196219-71478d80-8fc7-11ea-9e4a-87611693c5c2.jpeg)

**Comment by bendndi** [comment]
@dmitrij176 The software I'm using (Stardrop)  uses implied hydrogens on heteroatoms (you'll notice that the hydroxyls are also missing hydrogens). Unfortnately I'm not able to change this setting.

**Comment by dmitrij176** [comment]
@Wilson-Lm would you be able to share any up to date biological data for my analogues ?

I need it for my upgrade report R&D section.

**Comment by Wilson-Lm** [comment]
Hey @dmitrij176 , 
Give me until the end of tomorrow and I will upload the up to date _in vitro _ data for all compounds we tested. I will fill them in the excel sheet we have on github. or do you prefer it if I email you an excel sheet?  The _in vivo_ data will have to wait a little bit longer as we're still currently running the experiments.

**Comment by dmitrij176** [comment]
Hi again @Wilson-Lm . Ok, thanks a lot. Upload whichever way you prefer, all options are suitable.

**Comment by dmitrij176** [comment]
Hi @Wilson-Lm. Any news on the _in vitro_ and _in vivo_ data?

**Comment by Wilson-Lm** [comment]
Hi @dmitrij176, I have updated the _in vitro_ data in the main file on github (my apologies for forgetting to tell you). The data is up to date. 
_In vivo_ data however is still not ready. I am running the last larvae experiment to re-confirm the hits we have found. I will be able to give an update on that in the next meeting. The weather has been quite forgiving lately and thus we are able to stretch out the larvae experiment this year. From experience in the past few years, normally we wont be able to do that in November. Fingers cross we'll get a clearer view on how the compounds fair in this last larvae run!

**Comment by dmitrij176** [comment]
> Hi @dmitrij176, I have updated the _in vitro_ data in the main file on github (my apologies for forgetting to tell you). The data is up to date.
> _In vivo_ data however is still not ready. I am running the last larvae experiment to re-confirm the hits we have found. I will be able to give an update on that in the next meeting. The weather has been quite forgiving lately and thus we are able to stretch out the larvae experiment this year. From experience in the past few years, normally we wont be able to do that in November. Fingers cross we'll get a clearer view on how the compounds fair in this last larvae run!

Thanks a lot. Do you think that youll be able to get all the _in vivo_ data by next meeting (scheduled on Nov 24th)? I just need to understand how to finish my conclusions in the upcoming upgrade report i.e with just _in vitro_  or together with larval assays.

**Comment by Wilson-Lm** [comment]
I 

> > Hi @dmitrij176, I have updated the _in vitro_ data in the main file on github (my apologies for forgetting to tell you). The data is up to date.
> > _In vivo_ data however is still not ready. I am running the last larvae experiment to re-confirm the hits we have found. I will be able to give an update on that in the next meeting. The weather has been quite forgiving lately and thus we are able to stretch out the larvae experiment this year. From experience in the past few years, normally we wont be able to do that in November. Fingers cross we'll get a clearer view on how the compounds fair in this last larvae run!
> 
> Thanks a lot. Do you think that youll be able to get all the _in vivo_ data by next meeting (scheduled on Nov 24th)? I just need to understand how to finish my conclusions in the upcoming upgrade report i.e with just _in vitro_ or together with larval assays.

I wont be able to give the full data as this last larvae experiment will run till 27th of November. However, I reckon the data that I will show will most likely represent how the end results look like. I am hoping that this last run wont give us a completely different data pattern.

**Comment by dmitrij176** [comment]
Hi @Wilson-Lm. Hope you had a good Christmas break. Just wanted to check with you if there are any news regarding _in vivo_ data from the latest assays? Will there be updates on the Master List?

**Comment by Wilson-Lm** [comment]
> Hi @Wilson-Lm. Hope you had a good Christmas break. Just wanted to check with you if there are any news regarding _in vivo_ data from the latest assays? Will there be updates on the Master List?

Hi Dimitrij, Happy New Year!
The newest data can be found in the spread sheet, in the MyOS compounds and Pandemic Box sheet. Log-Rank and % survival data was updated. 
For the MyOS compounds, 1 compound - DM7-1 significantly increased survival, and in the Pandemic Box, we found 4 that increased survival - Olorofim, Fenbendazole, MMV1782387, Ravunonazole. 
I have the LogD data for pandemic box compounds but not for the fenarimol analogues.

**Comment by dmitrij176** [comment]
Hi Hung (@fantasy121 ). Hope all is going well in Sydney. I have assigned MYOS codes to novel Fenarimols in the Master List (https://docs.google.com/spreadsheets/d/1YhK-3i2KwuVabo1GbZSgVjAUbavEICCMKi5v-EhNq80/edit#gid=954203120) and need to reconfirm some details with you. I came across some molecules of yours which either did not have SMILES or there was an uncertainty regarding the batch number (due to your internal coding). To simplify things, I want to provide you with specific details which would help to navigate you. All unresolved Fenarimols are highlighted in red.
Summary:
**Row 5**: EPL-BS0495. I've put it as a second batch (last two digits in the MYOS code) but wasnt sure, since the same molecule appeared in row 3 (assigned as first batch). 
**Rows 16, 33, 34**: Batch issue.
**Rows 134-138**: SMILES missing, so could not reconfirm the structure. Ive put them as free bases. Just check please.

If you have any questions, let me know.
Thanks

**Comment by fantasy121** [comment]
@dmitrij176, all of my fenarimol analogues should appear as alcohols (OH) and not free bases (O-). I think the missing H is an artefact of the digital text-to-structure rendering of SMILES strings. As my work up is water, all O- should be protonated.

The F or P notations following my internal codes refer to the fractions that the desired compounds were eluted from the column. F notation was used for all compounds in 2017 (HPD01-X to HPD30-X) and P notation was used for all current molecules (HPD031-X onwards). My current notations have also added an extra zero in the hundred position to accomodate for future compounds (HPD30-X old system vs HPD031-X new system).

As you have come across HPD28-1 (4-fold racemate) where the structure is as follows:
[
![20200118 HPD28-1 diastereomer](https://user-images.githubusercontent.com/36176821/104871545-74d94400-599f-11eb-8241-b62dadc62d23.jpeg)
](url)
I resolved the two pairs of isomers:
F2 is RR and SS isomer pair(themselves are still 2-fold racemic)
F3 is the RS and SR isomer pair (themselves are still 2-fold racemic)

HPD28-1F2 and HPD28-1F3 are distinct isolated novel products and must not be merged. I think their isomer identity got left behind when they were appropriated into the spreadsheet. I will go ahead and add that info back.

This is why I now include fractions as a suffix (F or P) for my codes, for absolute clarity, and to avoid confusion when I happen to get out multiple desired products in one reactions. Please keep the F/P suffix so we can refer to my work in LabArchives and thesis if necessary.

Confirm Row 3 is synthesised by me because that is how I label my resynthesis compound in the ID column where I put my internal code first (HPD14-1), then the compound I'm trying to resynthesise (EPL-BS0495).

Row 5 is most likely acquired from the Epichem library (you can see in the "by whom" column my name is not there. @wwjvdsande @Wilson-Lm @mattodd can you double check this?

I think the original purpose of row 3 vs row 5 is to see whether I could replicate the efficacy of the Epichem compound.

Yes to row 16 HPD17-2F3. Keep as is, product is from eluted fraction 3 of the repeated reaction HPD17-2.

I have put the SMILES strings in for Row 134 to 138.
NB: Row 134 is not EPL-BS0495, not sure why it says so (FYI EPL-BS0495 corresponds to HPD14-1 on rows 3 and 5. HPD14-1 is my attempt at resynthesising EPL-BS0495). Row 134 (HPD033-2P4) is a novel compound. I have removed this inaccurate piece of information from row 134.

For future references, if you need a quick double check of data for my compounds, or if you're in a rush and need to access a snapshot of everything I did the the lab (and where I'm not there to answer right away) you can consult this page which should have most of the relevant things pertaining to the molecules I'm making including structures, names, my codes and SMLES. Hope this helps. https://au-mynotebook.labarchives.com/share/Hung%2520Duong%2520-%2520PhD/Mi42fDY4MTUwLzIvVHJlZU5vZGUvNDczMzQ5ODkyfDYuNg==

**Comment by dmitrij176** [comment]
> @dmitrij176, all of my alcohol fenarimol analogues should appear as protonated alcohols (OH) and not free bases (O-). I think the missing H is an artefact of the digital text-to-structure rendering of SMILES strings. As my work up is water, all O- should be protonated.
> 
> The F or P notations following my internal codes refer to the fractions that the desired compounds were eluted from the column. F notation was used for all compounds in 2017 (HPD01-X to HPD30-X) and P notation was used for all current molecules (HPD031-X onwards). For my current notations, I have also added an extra zero in the hundred position to accomodate for future compounds (HPD30-X old system vs HPD031-X new system).
> 
> As you have come across HPD28-1 (4-fold racemate) where the structure is as follows:
> [ ![20200118 HPD28-1 diastereomer](https://user-images.githubusercontent.com/36176821/104871545-74d94400-599f-11eb-8241-b62dadc62d23.jpeg) ](url)
> I resolved the two pairs of isomers:
> F2 is RR and SS isomer pair(themselves are still 2-fold racemic)
> F3 is the RS and SR isomer pair (themselves are still 2-fold racemic)
> 
> HPD28-1F2 and HPD28-1F3 are distinct isolated novel products and must not be merged. I think their isomer identity got left behind when they were appropriated into the spreadsheet. I will go ahead and add that info back. I also updated new SMILES strings to accurately reflect the isomers
> 
> This is why I now include fractions as a suffix (F or P) for my codes, for absolute clarity, and to avoid confusion when I happen to get out multiple desired products in one reaction. Please keep the F/P suffix so we can refer to my work in LabArchives and thesis if necessary.
> 
> Confirm Row 3 is synthesised by me because that is how I label my resynthesis compound in the ID column where I put my internal code first (HPD14-1), then the compound I'm trying to resynthesise (EPL-BS0495).
> 
> Row 5 is most likely acquired from the Epichem library (you can see in the "by whom" column my name is not there). @wwjvdsande @Wilson-Lm @mattodd can you double check this?
> 
> I think the original purpose of row 3 vs row 5 is to see whether I could replicate the efficacy of the Epichem compound.
> 
> Yes to row 16 HPD17-2F3. Keep as is, product is from eluted fraction 3 of the repeated reaction HPD17-2.
> 
> I have put the SMILES strings in for Row 134 to 138.
> NB: Row 134 is not EPL-BS0495, not sure why it says so (as clarified above, EPL-BS0495 corresponds to HPD14-1 on rows 3 and 5. HPD14-1 is my attempt at resynthesising EPL-BS0495). Row 134 (HPD033-2P4) is a novel compound. I have removed this inaccurate piece of information from row 134.
> 
> For future references, if you need a quick double check of data for my compounds, or if you're in a rush and need to access a snapshot of everything I did the the lab (and where I'm not there to answer right away) you can consult this page which should have most of the relevant things pertaining to the molecules I'm making including structures, names, my codes and SMLES. Hope this helps. https://au-mynotebook.labarchives.com/share/Hung%2520Duong%2520-%2520PhD/Mi42fDY4MTUwLzIvVHJlZU5vZGUvNDczMzQ5ODkyfDYuNg==
> 
> Hope this helps

Yes, thanks Hung (@fantasy121).
To be fair, I actually visited your Labarchive, but the Honours version as some of the mentioned analogs refer to legacy database. Anyway, all is sorted now, so no problems. 
Indeed, stereochemistry needs to be taken into account. Same refers to formulations (bases, salts, alcohols etc.). 
I will update everything accordingly.
